hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/XceiverServerRatis.java

```diff
@@ -630,6 +640,10 @@ void handleInstallSnapshotFromLeader(RaftGroupId groupId,
     handlePipelineFailure(groupId, roleInfoProto);
   }
 
+  @VisibleForTesting
+  public boolean isClosed() {
```

Comment:
lets remove this. and this can be replaced with

raftServer.getServer().getLifeCycleState().isClosingOrClosed()